config-example/admins.txt

```diff
@@ -33,6 +33,6 @@ tastyfish - Retired Admin
 uristqwerty - Game Master
 wrongnumber - Game Admin
 mij - Game Admin
-
+germansanta - Game Admin
```

Comment:
based

Reply:
would be, if it wasn't merging an example text lmao